src/pages/Strategies/components/StrategyCard/StrategyCard.tsx

```diff
@@ -1,56 +1,53 @@
+import Big from 'big.js';
+import { ReactNode } from 'react';
 import { useTranslation } from 'react-i18next';
-import { Link } from 'react-router-dom';
 
 import { formatPercentage } from '@/common/utils/utils';
 import { Card, CardProps, CoinIcon, Flex, H1, P, Strong } from '@/component-library';
-import { STRATEGIES, StrategyType } from '@/types/strategies';
-import { PAGES } from '@/utils/constants/links';
 
-import { useGetStrategyInsights } from '../../hooks/use-get-strategy-insights';
+import { StrategyRisk } from '../../types';
 import { StrategyTag } from '../StrategyTag';
 import { StyledEarningCard, StyledEarnSection } from './StrategyCard.style';
 
 type Props = {
-  strategyType: StrategyType;
+  title: ReactNode;
+  description: ReactNode;
+  ticker: string;
+  risk: StrategyRisk;
+  interestRate: Big;
 };
 
 type InheritAttrs = Omit<CardProps, keyof Props>;
 
 type StrategyCardProps = Props & InheritAttrs;
 
-const StrategyCard = ({ strategyType, ...props }: StrategyCardProps): JSX.Element => {
+const StrategyCard = ({ title, description, ticker, risk, interestRate, ...props }: StrategyCardProps): JSX.Element => {
   const { t } = useTranslation();
-  const { interest } = useGetStrategyInsights(strategyType);
-  const { descriptionCard, title, currency, risk, path, tags } = STRATEGIES[strategyType];
```

Comment:
We wont have more than 2 tags and one of the tags only shows when the risk is low